client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/hooks.tsx

```diff
@@ -16,17 +17,24 @@ const createDraftLine = (
   supplyQuantity: line.supplyQuantity,
 });
 
-export const useDraftRequisitionLine = (line: ResponseLineFragment) => {
-  const { id: reqId } = useResponse.document.fields('id');
+export const useDraftRequisitionLine = (item?: ItemRowFragment | null) => {
+  const { id: reqId, lines } = useResponse.document.fields(['id', 'lines']);
```

Comment:
Using existing pattern for now, but this could be updated straight from endpoint instead of going through a draft